docs/en/siem/machine-learning.asciidoc

```diff
@@ -21,35 +21,185 @@ image::ml-ui.png[]
+or `winlogbeat-*`) via {kib} -> Management -> Index Patterns, the jobs will be installed
+on page load, and will be displayed within the `Anomaly Detection` interface.
+
+NOTE: Machine learning jobs look back and analyse two weeks of historical data prior to
+the time they are enabled. After jobs are enabled, they continuously analyse incoming data.
+When jobs are stopped and restarted within the two week timeframe, previously
+analysed data is not processed again.
+
+The following prebuilt jobs are available:
+[horizontal]
+rare_process_by_host_windows_ecs::
++
+rare_process_by_host_linux_ecs:: Identifies rare processes that do not usually run
+on individual Windows/Linux hosts, which can indicate execution of unauthorized
+services, malware, or persistence mechanisms.
++
+Processes are considered rare when they only run occasionally as compared with
+other processes running on the host.
++
+Beats required on hosts:
+
+* Auditbeat (Linux)
+* Winlogbeat (Windows)
+windows_anomalous_network_activity_ecs::
++
+linux_anomalous_network_activity_ecs:: Identifies Windows/Linux processes that do
+not usually use the network but have unexpected network activity, which could indicate
+command-and-control, lateral movement, persistence, or data exfiltration activity.
++
+A process with unusual network activity can denote process exploitation or injection,
+where the process is used to run persistence mechanisms that allow a malicious actor
+remote access or control of the host, data exfiltration, and execution of unauthorized
+network applications.
++
+Beats required on hosts:
+
+* Auditbeat (Linux)
+* Winlogbeat (Windows)
+* Packetbeat (Windows and Linux)
+windows_anomalous_path_activity_ecs:: Identifies processes started from atypical
+folders in the file system, which might indicate malware execution or persistence
+mechanisms.
++
+In corporate Windows environments, software installation is centrally managed and
+it is unusual for programs to be executed from user or temporary directories.
+Processes executed from these locations can denote that a user downloaded software
+directly from the internet or a malicious script/macro executed malware.
++
+Beats required on hosts:
+
+* Winlogbeat
+windows_anomalous_process_all_hosts_ecs::
++
+linux_anomalous_process_all_hosts_ecs:: Searches for rare processes running on
+multiple Windows/Linux hosts in an entire fleet or network.
++
+This reduces the detection of false positives since automated maintenance processes
+often only occasionally run on a single machine but are common to all or many hosts
+in a fleet.
++
+Beats required on hosts:
+
+* Auditbeat (Linux)
+* Winlogbeat (Windows)
+windows_anomalous_process_creation:: Identifies unusual parent/child process
+relationships that could indicate malware execution or persistence mechanisms.
++
+Malicious scripts often call on other applications and processes as part of their
+exploit payload. For example, a malicious Office document runs a script that starts
```

Comment:
original was "For example, when a malicious Office document runs scripts as part of an exploit payload, Excel or Word may start a script interpreter process, which runs a script that downloads and runs malware."

Reply:
Corrected